Classes/Command/TradosCommandController.php

```diff
@@ -40,25 +41,33 @@ class TradosCommandController extends CommandController
      *
      * This command exports a specific site including all content into an XML format.
      *
-     * @param string $startingPoint The node with which to start the export, relative to the site node. Optional.
+     * @param string $startingPoint The node with which to start the export: as identifier or the path relative to the site node.
      * @param string $sourceLanguage The language to use as base for the export.
-     * @param string $targetLanguage The target language for the translation, optional.
-     * @param string $filename Path and filename to the XML file to create.
-     * @param string $modifiedAfter
+     * @param string|null $targetLanguage The target language for the translation, optional.
+     * @param string|null $filename Path and filename to the XML file to create.
+     * @param string|null $modifiedAfter
      * @param boolean $ignoreHidden
      * @return void
+     * @throws \Exception
      */
-    public function exportCommand($startingPoint, $sourceLanguage, $targetLanguage = null, $filename = null, $modifiedAfter = null, $ignoreHidden = true)
+    public function exportCommand(string $startingPoint,
+                                  string $sourceLanguage,
+                                  string $targetLanguage = null,
+                                  string $filename = null,
+                                  string $modifiedAfter = null,
+                                  bool $ignoreHidden = true)
     {
```

Comment:
It's both, I guess. I don't do that, unless it's really long (and in this case I don't consider it long.) And in the Neos codebase you'll find that rarely (don't know an example off the top of my head.) Sorry for the vague answer…

That being said, this could be done if sticking to the PSR-2/PSR-12 rules:
    
```suggestion
    public function exportCommand(
        string $startingPoint,
        string $sourceLanguage,
        string $targetLanguage = null,
        string $filename = null,
        string $modifiedAfter = null,
        bool $ignoreHidden = true
    ) {
```